massgov/openmass
# Replace Texta with Noto in admin theme

The amin theme is still loading Texta fonts. 
Because Texta fonts being loaded and some font-family css are still pointing to Texta, part of the page is rendered with Texta.
Noto Sans are not loaded, so the elements that are set to use Noto Sans are falling back to Helvetica
 https://edit.mass.gov/admin/content
![Screen Shot 2020-10-28 at 1 06 23 PM](https://user-images.githubusercontent.com/5789411/97470977-9685da80-191e-11eb-91ae-e42734f93a05.png)


To test:
- Build the site and navigate to an edit page
- Inspect and open the network tab
- Make sure you don't see Texta fonts being loaded, instead Noto fonts are being loaded 
- Inspect elements on the page and make sure the font-family is set to Noto instead of Texta
